### PR TITLE
[8.0][MIG] stock_picking_send_by_mail: Backport from 9.0

### PR DESCRIPTION
**Pull request by quentinDupont**
Backport from 9.0 to 8.0

### CODE REVIEW

File: stock_picking_send_by_mail/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 - Sandra Figueroa Varela
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking by Mail",
+    "summary": "Send stock picking by email",
+    "version": "8.0.1.0.0",
```
Comment (Nikul-Chaudhary) [inline stock_picking_send_by_mail/__openerp__.py:9]: @quentinDupont Add website key : `https://github.com/OCA/stock-logistics-workflow`

File: stock_picking_send_by_mail/data/stock_picking_mail.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                Web :&nbsp;<a href="${object.company_id.website}">${object.company_id.website}</a>
+            </div>
+        %endif
+        <p></p>
+    </div>
+</div>
+            ]]></field>
+        </record>
+    </data>
+</openerp>
```
Comment (Nikul-Chaudhary) [inline stock_picking_send_by_mail/data/stock_picking_mail.xml]: and new blank line

File: stock_picking_send_by_mail/views/stock_picking_view.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            <field name="arch" type="xml">
+                <field name="state" position="before">
+                    <button name="action_picking_send" string="Send by Email"
+                            type="object" attrs="{'invisible': [('picking_type_code', '!=', 'outgoing')]}"
+                            groups="base.group_user"/>
+                </field>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (Nikul-Chaudhary) [inline stock_picking_send_by_mail/views/stock_picking_view.xml]: also here

**Review by Nikul-Chaudhary — CHANGES_REQUESTED** [review]
Improve Code

**Comment by quentinDupont** [comment]
> Improve Code

Thanks ! Here is the new commit with the changes.
Do i need to squash the commits ?

**Comment by rousseldenis** [comment]
@quentinDupont Yes and you can rename IMHO the commit to "[8.0][MIG] stock_picking_send_by_mail: Backport from 9.0"

**Comment by rousseldenis** [comment]
@quentinDupont Could you check travis?

**Comment by Nikul-Chaudhary** [comment]
> > Improve Code
> 
> Thanks ! Here is the new commit with the changes.
> Do i need to squash the commits ?

Yes, Need to squashed

**Comment by quentinDupont** [comment]
> @quentinDupont Could you check travis?

Ok so i check travis, made the changes to be green.
Now i have an error i don't understand.

 ```
fatal: unable to access 'https://github.com/odoo/odoo.git/': transfer closed with outstanding read data remaining
The command "travis_install_nightly" failed and exited with 128 during .
```

Can I do something about it ?

**Comment by rousseldenis** [comment]
> Ok so i check travis, made the changes to be green.
> Now i have an error i don't understand.
> 
> ```
> fatal: unable to access 'https://github.com/odoo/odoo.git/': transfer closed with outstanding read data remaining
> The command "travis_install_nightly" failed and exited with 128 during .
> ```
> Can I do something about it ?

I see that in other PR's too. Will ask. Maybe related to 8.0 branch

**Comment by quentinDupont** [comment]
> I see that in other PR's too. Will ask. Maybe related to 8.0 branch

Thank you !

**Comment by quentinDupont** [comment]
Ok Travis went well this time. I squash my commits. Everything is fine now :)

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by tbaden — APPROVED** [review]
code review

@Nikul-Chaudhary the requested changes were done

**Pull request merged**